src/django/oar/urls.py

```diff
@@ -135,4 +135,9 @@
     path('api/geocoder/', views.get_geocoding, name='get_geocoding'),
 ] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
 
-urlpatterns = public_apis + internal_apis + info_apis
+api_v1 = [
+     path('api/v1/production-locations',
+          views.ProductionLocations.as_view(), name='production_locations')
```

Comment:
Please, use absolute import for the `ProductionLocations` view. It will make the location of the class more obvious.

Reply:
Yes, as I recall from the discussion with Peter, we are moving away from this approach.